spritesh.gemspec

```diff
@@ -0,0 +1,12 @@
+Gem::Specification.new do |s|
+  s.name        = "spritesh"
+  s.version     = "1.0.7"
+  s.summary     = "Build an SVG sprite from a folder of SVG files."
+  s.description = "A shell script to build an SVG sprite from a folder of SVG files."
+  s.authors     = ["Cade Scroggins"]
+  s.email       = "hello@cadejs.com"
+  s.files       = ["bin/gem/spritesh", "bin/gem/spritesh.sh"]
```

Comment:
I might be able to read the package.json file and update the gemspec dynamically. It's just a ruby script, after all.